taskcluster/ci/build/windows.yml

```diff
@@ -40,22 +40,6 @@ task-defaults:
         exec-with: powershell
         command: taskcluster/scripts/build/windows.ps1
 
-
-windows/msvc:
-    description: "Windows Build"
-    treeherder:
-        symbol: B(msvc)
-    worker:
-        artifacts:
-            - type: file
-              name: public/build/Mozilla VPN.pdb
-              path: artifacts/Mozilla VPN.pdb
-    scopes:
-        by-level:
-            "3":
-                - secrets:get:project/mozillavpn/tokens
-            default: []
-
```

Comment:
Nit/Not a problem, just a cleanup: Since there's only 1 task type left in this kind, it would be good to merge `task-defaults` into`windows/opt`. Unless if we plan on adding another task?